.github/workflows/release.yaml

```
@@ -1,4 +1,4 @@
-name: Build Release
+name: Release
```

How does this sound?

```suggestion
name: Atomic Build and Release
```